client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -155,12 +130,12 @@ const InteractionEventForm : React.FC = () : JSX.Element => {
                         <HospitalEventForm/>
                     </Collapse>
                 }
-                {
+                {/* {
```

Comment:
This is currently on comment due to ux changes that I want to commit in a different branch

Reply:
cool